### PR TITLE
versions that should support v1

### PR DESCRIPTION
**Pull request by kuisathaverat**
IIUC all latest agent versions should be compatible with APM Server 6.5, nonetheless only older versions (v1) should work with APM Server 6.2, 6.3, and 6.4

Go - No v1 versions

Java - No v1 versions

Node.js 1
* [X] APM Server 6.7
* [X] APM Server 6.6
* [X] APM Server 6.5
* [x] APM Server 6.4
* [x] APM Server 6.3
* [x] APM Server 6.2

Python 3.0
* [X] APM Server 6.7
* [X] APM Server 6.6
* [X] APM Server 6.5
* [x] APM Server 6.4
* [x] APM Server 6.3
* [x] APM Server 6.2

Python 2.0
* [X] APM Server 6.7
* [X] APM Server 6.6
* [X] APM Server 6.5
* [x] APM Server 6.4
* [x] APM Server 6.3
* [x] APM Server 6.2

Ruby 1.x
* [X] APM Server 6.7
* [X] APM Server 6.6
* [X] APM Server 6.5
* [x] APM Server 6.4
* [x] APM Server 6.3
* [x] APM Server 6.2

RUM 1.x
* [X] APM Server 6.7
* [X] APM Server 6.6
* [X] APM Server 6.5
* [X] APM Server 6.4
* [ ] APM Server 6.3
* [ ] APM Server 6.2

### CODE REVIEW

File: tests/versions/apm_server.yml [hunk 1]
```diff
@@ -1,7 +1,5 @@
 APM_SERVER:
-  - '6.x'
-  - '6.6;--release'
   - '6.5;--release'
-#  - '6.4;--release'
-#  - '6.3;--release'
-#  - '6.2;--release'
+  - '6.4;--release'
+  - '6.3;--release'
+  - '6.2;--release'
```
Comment (simitt) [inline tests/versions/apm_server.yml]: APM Server `6.6` and `6.7` also support `v1`, they should be added.

**Comment by axw** [comment]
Correct, the Go agent only supports 6.5+

File: tests/versions/go.yml [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
   - GO_AGENT: 'github;master'
     APM_SERVER: '6.3;--release'
   - GO_AGENT: 'github;master'
     APM_SERVER: '6.2;--release'
+  - GO_AGENT: 'github;v1.2.0'
+    APM_SERVER: '6.4;--release'
+  - GO_AGENT: 'github;v1.2.0'
+    APM_SERVER: '6.3;--release'
+  - GO_AGENT: 'github;v1.2.0'
+    APM_SERVER: '6.2;--release'
```
Comment (simitt) [inline tests/versions/go.yml]: The go agent does not support v1, so it should be completely removed from this branch.

File: tests/versions/java.yml [hunk 3]
```diff
@@ -3,24 +3,12 @@ JAVA_AGENT:
...
     APM_SERVER: '6.2;--release'
-  - JAVA_AGENT: 'github;master'
-    APM_SERVER: '6.7'
-  - JAVA_AGENT: 'github;master'
-    APM_SERVER: '6.6;--release'
-  - JAVA_AGENT: 'github;master'
-    APM_SERVER: '6.5;--release'
   - JAVA_AGENT: 'github;master'
     APM_SERVER: '6.4;--release'
   - JAVA_AGENT: 'github;master'
```
Comment (simitt) [inline tests/versions/java.yml]: Same as go agent, the java agent does not support v1 with any GA version, so should be removed from this branch.

File: tests/versions/nodejs.yml [hunk 4]
```diff
@@ -6,24 +6,12 @@ NODEJS_AGENT:
   - 'release;latest'
```
Comment (simitt) [inline tests/versions/nodejs.yml]: Only `release;1` is working with v1, so please remove the other agent versions.

File: tests/versions/python.yml [hunk 5]
```diff
@@ -5,26 +5,15 @@ PYTHON_AGENT:
   - 'github;master'
   - 'release;latest'
   - 'release;3.0'
+  - 'release;2.0'
```
Comment (simitt) [inline tests/versions/python.yml]: Please remove `master` and `release:latest`

File: tests/versions/ruby.yml [hunk 6]
```diff
@@ -7,24 +7,12 @@ RUBY_AGENT:
   - 'release;latest'
```
Comment (simitt) [inline tests/versions/ruby.yml]: Only ruby 1.x

File: tests/versions/apm_server.yml (hunk 1 above)
Comment (kuisathaverat) [reply tests/versions/apm_server.yml]: added, I have supposed that every agent that runs on 6.5 runs on 6.6 and 6.7

File: tests/versions/apm_server.yml [hunk 7]
```diff
@@ -1,7 +1,7 @@
 APM_SERVER:
-  - '6.x'
+  - '6.7;--release'
```
Comment (simitt) [inline tests/versions/apm_server.yml]: 6.7 is not released yet, you will need to use the github branch (by defining `6.7`)

**Pull request merged**